Resources/Prototypes/Parkstation/Entities/Objects/Devices/pinpointer.yml

```diff
@@ -0,0 +1,37 @@
+- type: entity
+  name: anomaly tracker
+  id: PinpointerAnomaly
+  parent: PinpointerBase
+  components:
+  - type: Pinpointer
+    component: Anomaly
+    mediumDistance: 24
+    closeDistance: 12
+    reachedDistance: 5
+    precision: 0.2
+  - type: Sprite
+    sprite: SimpleStation14/Objects/Devices/pinpointer.rsi
+    layers:
+    - state: pinpointer_anomaly
+      map: ["enum.PinpointerLayers.Base"]
+    - state: pinonnull
+      map: ["enum.PinpointerLayers.Screen"]
```

Comment:
There is an anomaly tracker now with a cool sprite, but I don't think it tells you where it is, just beeps faster and faster when near.